VisualPinball.Unity/VisualPinball.Unity/VPT/Gate/GateBehavior.cs

```diff
@@ -18,5 +19,36 @@ protected override Engine.VPT.Gate.Gate GetItem()
 		{
 			return new Engine.VPT.Gate.Gate(data);
 		}
+
+		public override ItemDataTransformType EditorPositionType => ItemDataTransformType.ThreeD;
+		public override Vector3 GetEditorPosition()
+		{
+			return data.Center.ToUnityVector3(data.Height);
+		}
+		public override void SetEditorPosition(Vector3 pos)
+		{
+			data.Center = pos.ToVertex2Dxy();
+			data.Height = pos.z;
+		}
+
+		public override ItemDataTransformType EditorRotationType => ItemDataTransformType.OneD;
+		public override Vector3 GetEditorRotation()
+		{
+			return new Vector3(data.Rotation, 0f, 0f);
+		}

```

Comment:
Just randomly commenting that you can also do `public override Vector3 GetEditorRotation() => new Vector3(data.Rotation, 0f, 0f)` for oneliners. No biggie but it gets more readable IMO.